windows/deploy_windows.ps1

```diff
@@ -1,9 +1,9 @@
 param (
     # Replace default path with system Qt installation folder if necessary
     [string] $QtInstallPath32 = "C:\Qt\5.15.2",
-    [string] $QtInstallPath64 = "C:\Qt\5.15.2",
+    [string] $QtInstallPath64 = "C:\Qt\6.8.1",
```

Comment:
This would change the local build environment. We'll need to update the COMPILING.md file probably.

Reply:
Not sure what you mean?

Reply:
As far as I know, changing this variable only affects where the script assumes Qt is if no parameters are set. The COMPILING.md file might be changed to tell the user that they should install multiple versions of Qt locally.

Reply:
```
### Supported platforms

We support the following platforms and versions:

- **Windows 10** or later
- **macOS 10.10** or later
- **Ubuntu 20.04** or later, **Debian 11** or later, most Linux flavors with recent enough Qt versions. (Currently, it may still build on Ubuntu 18.04 and Debian 10, but the binaries built by Github will not run on those versions.)

_While Android and iOS aren't officially supported, please don't break their builds._

Please try to avoid breaking any build by introducing platform-specific code. Check the Github builds all worked before raising a pull request.
Check to see if any newly introduced Qt calls, parameters, properties or constants are available in the minimum supported Qt version, which is currently **5.12.2**. Note that code _style_ in a file may be Qt 4.x. While you should normally stick to existing style, if you make large-scale modifications, updating to Qt 5.12.2 style is recommended.
Maintain C++11 compatibility throughout the code.
```
This doesn't mention _where_ Qt should be installed.  It also states 5.12.2 is the _minimum_ version: which isn't true for several platforms for several releases, anyway.  So I don't think the change here conflicts with it.

Reply:
OK, I've made `COMPILING.md` reference the deploy script rather than duplicating information that needs maintaining in multiple places (along with giving an example).